pkgs/os-specific/linux/firmware/fwupd/add-option-for-installation-sysconfdir.patch

```diff
@@ -1,36 +1,8 @@
-From 44887227f7f617cbf84713ec45685cb4999039ff Mon Sep 17 00:00:00 2001
-From: Jan Tojnar <jtojnar@gmail.com>
-Date: Tue, 30 Oct 2018 22:26:30 +0100
-Subject: [PATCH] build: Add option for installation sysconfdir
```

Comment:
Don't remove Jan's great commit msg :rofl:

Reply:
Just an artifact of how I regenerated it, haha, we can keep it for sure sorry :).